docs/DeveloperGuide.adoc

```diff
@@ -46,6 +46,7 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 |`* * *` |user |add a new person |
 |`* * *` |user |delete a person |remove entries that I no longer need
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |user |edit a person's details |have updated details of people, without deleting the old contact and creating a new contact with updated details
```

Comment:
In this user story, this is good. However, you can try to be more specific when you write user stories for your project